cmd/main.go

```diff
@@ -28,10 +28,6 @@ import (
 	mountmanager "sigs.k8s.io/gcp-compute-persistent-disk-csi-driver/pkg/mount-manager"
 )
 
-func init() {
```

Comment:
we had two init functions before, I don't know if that was causing issues but it doesn't hurt to fix